gateway-server/src/main/java/org/apache/knox/gateway/util/KnoxCLI.java

```diff
@@ -275,6 +277,12 @@ private int init(String[] args) throws IOException {
           printKnoxShellUsage();
           return -1;
         }
+      } else if (args[i].equals("create-list-aliases")) {
```

Comment:
Does this need to be a separate command, or could it be another option to the existing create-alias(es) commands?

Reply:
It could be the same command. My thinking was that we are introducing two new functionalities which in my opinion could warrant a new command.

1. With this change we are able to add multiple aliases to multiple topologies. 
2. When the alias creation is done for a specific topology it will list all of the aliases for it.

Reply:
If the new functionality can be added as an option(s) to the existing command without changing the previous behavior of the existing command (without those options), then I think I would prefer this over an entirely separate command. Ultimately, we're just creating aliases.

Reply:
I moved the new functionalities into the existing command. This modification is not backward compatible because previously the --cluster parameter's position didn't matter, however from now on an alias will be created for a cluster if it is listed before the cluster.